packages/execution-environments/src/iframe/IFrameController.ts

```diff
@@ -0,0 +1,30 @@
+import ObjectMultiplex from '@metamask/object-multiplex';
+import { WindowPostMessageStream } from '@metamask/post-message-stream';
+import pump from 'pump';
+import { BaseController } from '../common/BaseControllers';
+import { SNAP_STREAM_NAMES } from '../common/enums';
+
+export class IFrameController extends BaseController {
```

Comment:
In line with the suggested renaming:

```suggestion
export class IFrameSnapExecutor extends BaseSnapExecutor {
```